e2e/testconfig/testconfig.go

```diff
@@ -166,6 +166,7 @@ func newDefaultSimappConfig(cc ChainConfig, name, chainID, denom string) ibc.Cha
 		},
 		Bin:            cc.Binary,
 		Bech32Prefix:   "cosmos",
+		CoinType:       "118",
```

Comment:
I wonder if there's a constant for this in the sdk or gaia.. we can leave it til later

Reply:
ah there is i think! i will use that one.